### PR TITLE
add spk-id & mix-up for xiaoice_noDP model(encoder outputs mix-up)

### PR DESCRIPTION
**Pull request by PeterGuoRuc**

### CODE REVIEW

File: egs/TEMPLATE/svs1/svs.sh [hunk 1]
```diff
@@ -290,6 +290,8 @@ if ! "${skip_data_prep}"; then
     if [ ${stage} -le 1 ] && [ ${stop_stage} -ge 1 ]; then
         log "Stage 1: Data preparation for data/${train_set}, data/${valid_set}, etc."
         # [Task dependent] Need to create data.sh for new corpus
+        echo ${local_data_opts}
+        echo ${fs}
```
Comment (ftshijt) [inline egs/TEMPLATE/svs1/svs.sh:294]: Can we remove it?

File: muskit/svs/xiaoice/XiaoiceSing.py [hunk 2]
```diff
@@ -42,6 +42,10 @@
 
 from muskit.torch_utils.nets_utils import pad_list
 
+import random
+from torch.distributions import Beta
+
+Beta_distribution = Beta(torch.tensor([0.5]), torch.tensor([0.5]))
```
Comment (ftshijt) [inline muskit/svs/xiaoice/XiaoiceSing.py:48]: Maybe introduce some arguments to it?

File: egs/TEMPLATE/svs1/svs.sh (hunk 1 above)
Comment (PeterGuoRuc) [reply egs/TEMPLATE/svs1/svs.sh]: I have removed it

File: muskit/svs/xiaoice/XiaoiceSing.py (hunk 2 above)
Comment (PeterGuoRuc) [reply muskit/svs/xiaoice/XiaoiceSing.py]: already add Fix Me Note

**Pull request merged**